api/invite_test.go

```diff
@@ -673,7 +673,7 @@ func TestInviteResponds(t *testing.T) {
 		{
 			desc:     "valid request to accept an team invite",
 			method:   http.MethodPut,
-			url:      fmt.Sprintf("/accept/team/invite/%s/%s", "123.456.789", "123456"),
+			url:      fmt.Sprintf("/accept/team/invite"),
```

Comment:
No need to use Sprintf without variables (I didn't suggest all other usage). 
```suggestion
			url:      "/accept/team/invite",
```